Sources/AppStoreConnectCLI/Commands/Profiles/CreateProfileCommand.swift

```diff
@@ -34,16 +34,6 @@ struct CreateProfileCommand: CommonParsableCommand {
     )
     var devicesUdids: [String]
 
-    func validate() throws {
-        if certificatesSerialNumbers.isEmpty {
-            throw ValidationError("Expected at least one certificate serial number.")
-        }
```

Comment:
Certs are required when creating a profile. Device identifiers are not.

Reply:
I have added certificates number validation back